astroid/brain/brain_attrs.py

```diff
@@ -10,16 +10,22 @@
 from astroid.nodes.node_classes import AnnAssign, Assign, AssignName, Call, Unknown
 from astroid.nodes.scoped_nodes import ClassDef
 
-ATTRIB_NAMES = frozenset(("attr.ib", "attrib", "attr.attrib", "attr.field", "field"))
+ATTRIB_NAMES = frozenset(
+    ("attr.ib", "attrib", "attr.attrib", "attr.field", "attrs.field", "field")
+)
 ATTRS_NAMES = frozenset(
     (
         "attr.s",
         "attrs",
+        "define",
```

Comment:
I wonder if we should add this.

Sadly this brain does not infer the actual function behind the decorator but only looks at the name. However, if anybody uses `define` as a decorator name in non-`attr` code they will now also get picked up by this brain. Fort some reason we allowed `attrib` and `field` here, see https://github.com/PyCQA/astroid/pull/1187#pullrequestreview-763579685.

Do we want to add `define` as well @Pierre-Sassoulas? I can't imagine it being a very often used name for decorators, but it might create troubles down the line..

Reply:
Well I don't like assuming that something is part of a lib only because of its name especially since we have inference capability in astroid. I know we're doing it for ``np`` in some place. But inference would permit to infer the real name of the origin lib. I don't know enough about brains to suggest something proper instead. What do you think @hippo91 ?

Reply:
@Pierre-Sassoulas i agree with you and @DanielNoord as well.
IMHO `define` should not be part of this set.

Reply:
okay, I've removed `@define` from the list of supported decorator names.